### ISSUE TITLE
author page - mills kelly - links with no accessible names

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#104]
https://rrchnm.org/author/mills-kelly/

The image links under "projects" on this page are showing as not having accessible names in ANDI. I also checked another author page and this alert comes up for the projects as well, so it's safe to assume this is an issue on all author pages. 
![Screen Shot 2020-10-06 at 4 38 42 PM](https://user-images.githubusercontent.com/618307/95257547-7b79ec00-07f2-11eb-8f5d-56d4e948f4e8.png)

**Comment by kimisgold** [comment]
Commit 1835bed should also take care of this issue. Also live and ready for another look.
